### PR TITLE
Fix violations of IDE0059

### PR DESCRIPTION
**Pull request by davidwengier**
Just removing some noise from the Error List.

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/ProjectRootImageProjectTreeModifierTests.cs [hunk 1]
```diff
@@ -157,12 +157,7 @@ public void CalculatePropertyValues_WhenSharedItemsImportFileAsTree_SetsIconToSh
 ")]
         public void CalculatePropertyValues_NonProjectRootAsTree_DoesNotSetIcon(string input)
         {
-            var imageProvider = IProjectImageProviderFactory.ImplementGetProjectImage(ProjectImageKey.ProjectRoot, new ProjectImageMoniker(new Guid("{A140CD9F-FF94-483C-87B1-9EF5BE9F469A}"), 1));
-
-            var propertiesProvider = CreateInstance(imageProvider);
-
             var tree = ProjectTreeParser.Parse(input);
-            var result = propertiesProvider.ChangePropertyValuesForEntireTree(tree.Children[0]);
```
Comment (davkean) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/ProjectRootImageProjectTreeModifierTests.cs]: The bug here is that the Assert should be checking "result" not "tree".

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/UnconfiguredProjectTasksServiceTests.cs [hunk 2]
```diff
@@ -101,7 +101,6 @@ public void UnloadCancellationToken_WhenUnderlyingUnloadCancellationTokenCancell
         {
             var source = new CancellationTokenSource();
             var tasksService = IProjectAsynchronousTasksServiceFactory.ImplementUnloadCancellationToken(source.Token);
-            var service = CreateInstance(tasksService);
 
             Assert.False(tasksService.UnloadCancellationToken.IsCancellationRequested);
```
Comment (davkean) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/UnconfiguredProjectTasksServiceTests.cs]: Bug here is that Assert should be checking "service".

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Properties/OutputTypeExValueProviderTests.cs [hunk 3]
```diff
@@ -52,8 +52,7 @@ public async Task SetValue(string incomingValue, string expectedOutputTypeValue)
                     SetValues = setValues
                 });
             var provider = new OutputTypeExValueProvider(properties);
-
-            var actualPropertyValue = await provider.OnSetPropertyValueAsync(incomingValue, null);
+            await provider.OnSetPropertyValueAsync(incomingValue, null);
             Assert.Equal(setValues.Single(), expectedOutputTypeValue);
```
Comment (davkean) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Properties/OutputTypeExValueProviderTests.cs]: Looks like it's checking the wrong result, I suspect "actualPropertyValue" is supposed to be in the assert.

Comment (davidwengier) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Properties/OutputTypeExValueProviderTests.cs]: Looks like `OnSetPropertyValueAsync` has a hardcoded null return, and this test has been broken in this way for a while, so the removal makes sense.

**Comment by davidwengier** [comment]
With respect to the commit stream that makes up this PR: This place is not a place of honor.. no highly esteemed deed is commemorated here... nothing valued is here.

😛

**Pull request merged**